src/components/autocompleter/autocompleter.test.jsx

```diff
@@ -0,0 +1,139 @@
+import React, { createRef } from 'react';
+import {
+  render, screen,
+} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import jestServer from '../../mocks/jest-server.js';
+import Autocompleter from './autocompleter.jsx';
+import mockHandlers from './mock-handlers.js';
+
+describe('src/components/autocompleter/autocompleter', () => {
+  beforeEach(() => {
+    jestServer.use(...mockHandlers());
+  });
+
+  const requiredProps = {
+    id: 'test-id',
+    label: 'Test label',
+    name: 'field-id',
```

Comment:
We might want to update this copy-pasta. This can be confusing since the `name` is not an ID. What about:
```suggestion
    name: 'field_name',
```